pages/Documents/DeveloperTools/LivePersonFunctions/Integrations/messaging-conversations.md

```diff
@@ -24,25 +24,14 @@ Along with the invocation, the function is sent a payload containing metadata re
 
 Functions for messaging listens for messaging events asynchronously. As a consequence this can cause race conditions with other parts of the platform. Therefore, it is considered best practice to use bots instead of Functions for implementing routing logic. Functions is a good option to sync data with third-party systems like CRMs or to save data in the [Context Warehouse](maven-context-warehouse-overview.html) in order to use it within Conversation Orchestrator. Routing via Functions makes sense whenever a conversation is in a stagnant state (i.e. not in process of being routed), e.g. a conversation is idle or a message line has been sent in off-hours. Otherwise, the asynchronous nature of its events might interfere with the proper flow of a "dynamic" conversation. 
 
-### Messaging events for Function Invocation
-
-Conversational Cloud messaging uses a series of "Conversation State Change Events" which get fired when specific actions or events occur within the conversation. You are able to use theses events to trigger functions within Functions.
-
-The following "Conversation State Change Events" can be used to trigger functions (see [Events](liveperson-functions-development-events-templates.html#events)):
 
-* New Conversation
+<div class="important">if the authenticated consumer's previous conversation was auto-closed, and the new one opened within 48 hours of that, the new conversation event won't be triggered.</div>
 
-* TTR (Time to Respond)
-
-* Participants Change
-
-* Conversation Idle
+### Messaging events for Function Invocation
 
-* Conversation Routing
+Conversational Cloud messaging uses a series of "Conversation State Change Events" which get fired when specific actions or events occur within the conversation. You are able to use theses events to trigger functions within Functions.
```

Comment:
Is there a link to the "Conversation State Change Events" for more specific information?

```suggestion
Conversational Cloud messaging uses a series of "Conversation State Change Events" which are fired when specific actions or events occur within the conversation. You are able to use theses events to trigger your functions.
```

Reply:
updated